samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -115,9 +115,70 @@ The following antenna switching patterns are possible:
 The application supports a maximum of 19 antennas in the direction finding mode.
 The RADIO can control up to 8 GPIO pins for the purpose of controlling the external antenna switches used in direction finding.
 
-The antenna is chosen by writing consecutive numbers to the SWITCHPATTERN register.
-This means that the antenna GPIO pins act like 8-bit registers.
-In other words, for the first antenna, antenna pin 1 is active, for the second antenna, pin 2 is active, for the third antenna, pins 1 and 2 are active, and so on.
+Antenna matrix configuration
+----------------------------
+
+To use this sample to test the Bluetooth Direction Finding feature, additional configuration of GPIOs is required to control the antenna array.
+Example of such configuration is provided in a devicetree overlay file :file:`nrf5340dk_nrf5340_cpunet.overlay`.
+
+The overlay file provides the information about which GPIOs should be used by the Radio peripheral to switch between antenna patches during the CTE reception or transmission.
```

Comment:
```suggestion
The overlay file provides the information of the GPIOs to be used by the Radio peripheral to switch between antenna patches during the CTE reception or transmission.
```
Should we open the abbreviation CTE?

Reply:
We can. I changed to Constant Tone Extension